app/controllers/admin/topics_controller.rb

```diff
@@ -40,7 +40,11 @@ def index
     if current_user.is_restricted? && teams?
       topics_raw = Topic.all.tagged_with(current_user.team_list, any: true)
     else
-      topics_raw = Topic
+      if params[:team].present?
```

Comment:
Yes thats a good solution.  That scope should have test coverage from the `acts_as_taggable_on` gem, but I should probably add one for the controller when params are present, so good idea there!